src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/GenerateCoverSheetService.java

```diff
@@ -99,11 +105,39 @@ private void prepareCoverSheet(CaseDetails caseDetails, String partyAddressCcdFi
                     : (Map) caseData.get(partyAddressCcdFieldName)))
                 .build();
             caseData.put(ADDRESSEE, addressee);
-            caseData.put(CTSC_CONTACT_DETAILS, buildCtscContactDetails());
+            caseData.put(COURT_CONTACT_DETAILS, getCourtContactDetails(caseDetails));
             caseData.put(CASE_NUMBER, nullToEmpty(caseDetails.getId()));
         }
     }
 
+    private String getCourtContactDetails(CaseDetails caseDetails) {
+        /*
+        if (isContestedApplication(caseDetails)) {
```

Comment:
this not ready?

Reply:
Not yet - will be doing it as a separate PR.
I want to make sure the consented journey is unaffected when my PR comes in. This way we can use a single method for populating both consented or contested court data on the template